src/renderers/webxr/WebXRManager.js

```diff
@@ -466,6 +463,8 @@ class WebXRManager extends EventDispatcher {
 						scaleFactor: framebufferScaleFactor
 					};
 
+					glBinding = new XRWebGLBinding( session, gl );
```

Comment:
Would it make sense to create a function that creates a binding if it doesn't exist yet and then returns it? That way we can use it in more places and don't have to worry about checking if it was already created.

Reply:
We already have getBinding in this file, so I guess we can put that logic there.
Note we're using getBinding in the aframe layer component
https://github.com/aframevr/aframe/blob/9ff61ffaf98c42b255fb17674eddb0d85c3239ba/src/components/layer.js#L262 to reuse that XRWebGLBinding instance.

Reply:
looks good to me!